recipes/jarvis-tools/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "jarvis-tools" %}
+{% set version = "2020.10.10" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/jarvis-tools-{{ version }}.tar.gz
+  sha256: b4cc45267100fe2cc4d114d1453ad39e87d35244ce511c44b1fc99ca277f8a07
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - joblib
+    - matplotlib-base
+    - numpy
+    - pytest
```

Comment:
That's a surprising dependency in `run`. I see that upstream has this in their setup.py as well. Is this really needed?

Reply:
It's been removed. Definitely not a run time dependency. Added an issue to have it removed, https://github.com/usnistgov/jarvis/issues/129

Reply:
Scrub that. It's listed in setup.py so is needed to build I assume.